udata/commands/db.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from os.path import join
+from pkg_resources import resource_isdir, resource_listdir, resource_string
+
+from flask import current_app
+
+from mongoengine.connection import get_db, DEFAULT_CONNECTION_NAME
+
+from udata.commands import submanager, green, yellow, cyan, purple
+
+log = logging.getLogger(__name__)
+
+
+m = submanager(
+    'db',
+    help='Databse related operations',
+    description='Handle all databse related operations and maintenance'
+)
+
+# A migration script wrapper recording the stdout lines
+SCRIPT_WRAPPER = '''
+function(plugin, filename, script) {{
```

Comment:
:)
It's the only way I found to keep existing migration, allow to keep writing JS migrations and to not depends on mongo client.